src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -582,6 +582,15 @@ public BoundExpression StaticCall(TypeSymbol receiver, MethodSymbol method, para
             return Call(null, method, args);
         }
 
+        public BoundExpression StaticCall(TypeSymbol receiver, WellKnownMember method, params BoundExpression[] args)
+        {
+            MethodSymbol methodSymbol = WellKnownMethod(method);
+            Debug.Assert(methodSymbol.IsStatic);
+            Debug.Assert(receiver.GetMembers(methodSymbol.Name).OfType<MethodSymbol>().Contains(methodSymbol));
+            Binder.ReportUseSiteDiagnostics(methodSymbol, Diagnostics, Syntax);
```

Comment:
I constructed the body of this method from a synthesis of the other StaticCall methods and some Property methods. The results looked kind of silly to me, but I wanted to make sure that I wasn't dropping any important checks. I'll streamline this to something rational.